keras_rs/src/layers/embedding/base_distributed_embedding.py

```diff
@@ -30,6 +30,19 @@ class DistributedEmbedding(keras.layers.Layer):
 
     ---
 
+    `DistributedEmbedding` is a layer optimized for TPU chips with SparseCore
+    and can dramatically improve the speed of embedding lookups and embedding
+    training, in particular for large embedding tables. It works by combining
+    multiple lookups in one invocations, and by sharding the embedding tables
+    across the available chips.
```

Comment:
Sharding doesn't necessarily make it faster on its own.  That leads to more communication for the collective, which can actually slow the lookup down if the table would otherwise fit on a single chip.  We might want to specify that it can make lookups faster for large tables that already need to be model-sharded.

Reply:
Changed wording and added a whole section about placement.